updateStableIds/src/main/java/org/reactome/release/update_stable_ids/UpdateStableIds.java

```diff
@@ -0,0 +1,170 @@
+package org.reactome.release.update_stable_ids;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+
+import static org.gk.model.ReactomeJavaConstants.*;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.persistence.TransactionsNotSupportedException;
+import org.gk.schema.GKSchemaClass;
+import org.gk.schema.InvalidAttributeException;
+import org.reactome.release.common.database.InstanceEditUtils;
+
```

Comment:
Can some of the unused imports be removed?

Reply:
Yea, I think that Eclipse tries to automatically remove them if they're not used, but its less-than-great at it

Reply:
Gotcha, I'll try to make that a habit before committing anything in the future